src/server/config.js

```diff
@@ -51,17 +36,14 @@ function loadBabelConfig(babelConfigPath) {
 export default function (configType, baseConfig, configDir) {
   const config = baseConfig;
 
-  // Search for a .babelrc in the config directory, then the module root
+  // Search for a .babelrc starting in the config directory
   // directory. If found, use that to extend webpack configurations.
-  let babelConfig = loadBabelConfig(path.resolve(configDir, '.babelrc'));
-  let inConfigDir = true;
-
-  if (!babelConfig) {
-    babelConfig = loadBabelConfig('.babelrc');
```

Comment:
Oh gotcha. Can `.storybook` be outside of the project? What would be the reason for it?

Because `find-babel-config` will make a search recursively, so if it's still inside the project, everything should work as expected.
If not, I can change the PR to match what was done before.